crates/aggchain-proof-core/src/full_execution_proof.rs

```diff
@@ -1,13 +1,21 @@
+use alloy_primitives::{Address, PrimitiveSignature, B256};
 use serde::{Deserialize, Serialize};
-#[cfg(target_os = "zkvm")]
 use sha2::{Digest as Sha256Digest, Sha256};
-use sp1_zkvm::lib::utils::words_to_bytes_le;
 
 use crate::{error::ProofError, keccak::digest::Digest, keccak::keccak256_combine};
 
+#[cfg(target_os = "zkvm")]
 type Vkey = [u32; 8];
 
-// Hardcoded for now, might see if we might need it as input
+/// Hardcoded hash of the "aggregation vkey".
+/// NOTE: Format being `hash_u32()` of the `SP1StarkVerifyingKey`.
+#[cfg(target_os = "zkvm")]
+pub const AGGREGATION_VKEY_HASH: Vkey = [0u32; 8]; // TODO: to put the right value
```

Comment:
In order to check the vkey hash before the proof checking (as done in #83), this constant would have to be available publicly. Either by exporting it from this crate or by moving it into some sort of `-common` crate. Also, it may be worth considering making the constant of the type `VKeyHash`, also introduced in #83.

I appreciate that if merging this PR is time sensitive, just exporting it from the crate as is might be the most viable option for now.